nifi/nifi-nar-bundles/nifi-solr-bundle/nifi-solr-processors/pom.xml

```diff
@@ -0,0 +1,92 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements. See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License. You may obtain a copy of the License at
+  http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.apache.nifi</groupId>
+        <artifactId>nifi-solr-bundle</artifactId>
+        <version>0.1.0-incubating-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>nifi-solr-processors</artifactId>
+    <packaging>jar</packaging>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.solr</groupId>
+            <artifactId>solr-solrj</artifactId>
+            <version>${solr.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-processor-utils</artifactId>
+        </dependency>
+        <!-- test dependencies -->
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-mock</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-simple</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>commons-logging</groupId>
+            <artifactId>commons-logging</artifactId>
+            <version>1.1.3</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.11</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.solr</groupId>
+            <artifactId>solr-core</artifactId>
+            <version>${solr.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <!-- These Lucene deps should be brought in through solr-core, but there
+         appears to be an issue with 5.0.0 that still references some 4.10.3 poms -->
```

Comment:
It looks like it still is, here are the dependencies brought in through solr-core 5.1.0:

+- org.apache.solr:solr-core:jar:5.1.0:test
[INFO] |  +- org.apache.lucene:lucene-analyzers-common:jar:4.10.3:test
[INFO] |  +- org.apache.lucene:lucene-analyzers-kuromoji:jar:5.1.0:test
[INFO] |  +- org.apache.lucene:lucene-analyzers-phonetic:jar:5.1.0:test
[INFO] |  +- org.apache.lucene:lucene-backward-codecs:jar:5.1.0:test
[INFO] |  +- org.apache.lucene:lucene-codecs:jar:5.1.0:test
[INFO] |  +- org.apache.lucene:lucene-core:jar:4.10.3:test
[INFO] |  +- org.apache.lucene:lucene-expressions:jar:5.1.0:test
[INFO] |  +- org.apache.lucene:lucene-grouping:jar:5.1.0:test
[INFO] |  +- org.apache.lucene:lucene-highlighter:jar:5.1.0:test
[INFO] |  +- org.apache.lucene:lucene-join:jar:5.1.0:test
[INFO] |  +- org.apache.lucene:lucene-memory:jar:5.1.0:test
[INFO] |  +- org.apache.lucene:lucene-misc:jar:5.1.0:test
[INFO] |  +- org.apache.lucene:lucene-queries:jar:5.1.0:test
[INFO] |  +- org.apache.lucene:lucene-queryparser:jar:4.10.3:test

I looked at the solr-core 5.1.0 pom, but I don't see why that is happening, and it looks like it got pulled from Maven central:

#Thu Apr 16 20:46:02 EDT 2015
solr-core-5.1.0.jar>central=
solr-core-5.1.0.pom>central=